docs/tutorial/typesfuns.rst

```diff
@@ -538,9 +538,10 @@ Note: Declaration Order and ``mutual`` blocks
 
 In general, functions and data types must be defined before use, since
 dependent types allow functions to appear as part of types, and their
-reduction behaviour to affect type checking. However, this restriction
-can be relaxed by using a ``mutual`` block, which allows data types
-and functions to be defined simultaneously:
+reduction behaviour to affect type checking (though this is only true of
```

Comment:
@ericqweinstein How about "and type checking can rely on how particular functions are defined" or something like that?